### PR TITLE
Make it compile on MSVC2010

### PR DESCRIPTION
**Pull request by stepanp**

### CODE REVIEW

File: tabbar.cpp [hunk 1]
```diff
@@ -51,7 +51,8 @@ TabBar::TabBar(QWidget *parent) :
 
     ProjectExplorer::SessionManager *sm = ProjectExplorer::SessionManager::instance();
     connect(sm, &ProjectExplorer::SessionManager::sessionLoaded, [this, em]() {
-        for (Core::DocumentModel::Entry *entry : Core::DocumentModel::entries())
+        Core::DocumentModel::Entry *entry = nullptr;
+        foreach (entry, Core::DocumentModel::entries())
```
Comment (trollixx) [inline tabbar.cpp]: What's the reason for out of loop scope `entry`? The following works with pre C++11 compilers.
```cpp
foreach (Core::DocumentModel::Entry *entry, Core::DocumentModel::entries())
```

File: tabbar.cpp [hunk 2]
```diff
@@ -110,7 +111,8 @@ void TabBar::addEditorTab(Core::IEditor *editor)
 void TabBar::removeEditorTabs(QList<Core::IEditor *> editors)
 {
     blockSignals(true); // Avoid calling activateEditor()
-    for (Core::IEditor *editor : editors) {
+    Core::IEditor *editor = nullptr;
+    foreach (editor, editors) {
```
Comment (trollixx) [inline tabbar.cpp]: Same as above.

**Comment by trollixx** [comment]
Surprisingly not that painful. Please, update according to my comments, and I shall merge it.

**Comment by trollixx** [comment]
Also, please change commit name to `Add MSVC 2010 support (fixes #10)`.

**Comment by stepanp** [comment]
Done :)

**Pull request merged**